lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -115,13 +118,29 @@ module.exports = function(self, options) {
             } else {
               info.portrait = true;
             }
+
+            if (info.extension === 'jpg' || info.extension === 'jpeg') {
```

Comment:
It can only be `jpg`. (We had an issue for a while where we didn't guarantee this but it was fixed some time ago. Tell me if I'm wrong...)